bootstrap/eksctl/eksctl.yaml

```diff
@@ -6,10 +6,10 @@ kind: ClusterConfig
 metadata:
   name: crossplane-blueprints
   region: us-west-2
-  version: "1.23"
+  version: "1.24"
 managedNodeGroups:
   - name: managed-ng-1
-    instanceType: m5.large
+    instanceType: t3.small
```

Comment:
This is not recommend. EMR on EKS example requires 1G memory and may not get scheduled with small overhead especially when users decide to install something on top to test. @csantanapr

Reply:
EMR on EKS? We don't have any example on this repo that would not work with 3 t3.small nodes

For an user to get started to learn crossplane and aws they don't need to create large nodes, the can start simple for the minimum to be use as an EKS control plane.

What about if they want to run Kubeflow, or Kafka, or EMR, or Cassandra? They need to adjust the cluster size for the workload they are running. 

But this is your repo I can revert it to m5 @nabuskey

Reply:
The composition we have in the repo for EMR on EKS will launch jobs into the same cluster so it's very possible pods may not be scheduled with a few daemon sets installed. I actually ran into it when I was writing the composition. Thanks for fixing it!
Also I don't consider this repository mine. We are all maintainers of this repo.